### PR TITLE
feat: work needed to bring in self contained styles

### PR DESCRIPTION
**Pull request by jbadan**
### Description

This will be a **breaking change** for consumers as they will need to update their webpack config as detailed in the updated readme.

This pr brings in the self-contained styles for Button, updates/simplifies the webpack configuration, updates jest config to allow for parsing scss.

Size-limit was not setup to handle scss files, I copied their internal config as much as possible with the addition of scss loaders. Caveat: the output now contains our peerDependencies (? I can't figure out why) and is reporting our **parsed size, not gzipped**

MASTER:
Screen Shot 2019-06-26 at 2 53 30 PM

This PR (note - peer dependencies like react-dom and react and some devdeps for webpack are now being included on the node_modules side - I've attempted to troubleshoot this for a while, but I think at this point it's out of range of the spike and should be a followup):
Screen Shot 2019-06-27 at 8 44 32 AM

Build:index works again! Thanks @greg-a-smith

### CODE REVIEW

File: .size-limit [hunk 1]
```diff
@@ -1,8 +1,7 @@
 [
   {
     name: "Fundamental-React Size",
-    webpack: true,
-    path: "lib/index.js",
-    limit: "50 KB"
+    config: "config/webpack.config.sizeLimit.js",
+    limit: "550 KB"
```
Comment (bcullman) [inline .size-limit]: whoa. sad.

Comment (jbadan) [reply .size-limit]: did you read the description? 😈 

50kb was gzipped, using the new custom webpack config I haven't been able to get it to return gzipped size to size-limit, so it is reporting the parsed size. While we're now at double the gzipped size (including the peer dependency weirdness).

We're really only 12kb larger than we were previously.

File: config/webpack.config.sizeLimit.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            },
+            {
+                test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+                use: [{
+                    loader: 'file-loader'
+                }]
+            }
+        ]
+    },
+    externals: {
```
Comment (jbadan) [inline config/webpack.config.sizeLimit.js]: Wooo! 🎉 Thanks @bcullman

File: src/Button/Button.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import '../../node_modules/fundamental-styles/components/button.scss';
```
Comment (greg-a-smith) [inline src/Button/Button.js]: Does this need to include the `../../node_modules` in the path or could the import be simplified to `import 'fundamental-styles/components/button.scss';`?

**Pull request merged**